### ISSUE TITLE
Library could not be loaded

### ISSUE THREAD

**Issue by miredirex** [werman/noise-suppression-for-voice#10]
**Operating system:** Windows 10 Pro
**Equalizer APO version:** 1.2

Steps to reproduce the issue: Add filter -> Plugins -> VST Plugins -> Choose any of the dll's of the latest release (Feb 13) -> "Library could not be loaded"
v0.1 (22 Jan 2018) works fine.

Yes, I've seen the "Windows plugins are untested." note but still would like to get a fix for that.

**Comment by werman** [comment]
Hi, thanks for the report, I'll see what I can do since I don't have windows nearby.

**Comment by werman** [comment]
Tested under Wine and saw:
> 0206:err:module:import_dll Library libwinpthread-1.dll (which is needed by L"build-mingw64\\bin\\vst\\librnnoise_vst.dll") not found

I have fixed this and updated the release. Could you test it?

**Comment by miredirex** [comment]
VST one works now. The other ones say 'Library does not contain needed functions.' but I think that's how it should be.

**Comment by werman** [comment]
Equalizer APO probably doesn't support ladspa and lv2 plugins so it's ok, thanks.

**Comment by h-a-s-k** [comment]
This is happening with latest rnnoise_stereo.dll but not with rnnoise_mono.dll

**Comment by Kaiman42** [comment]
I'm getting the same "library does not contain needed functions." error and I've tried using different versions and types of rnnoise. rnnoise_ladspa, rnnoise_mono 1 and 2, rnnoise_stereo 1 and 2, rnnoise.vst3. Of all the versions of you released, I downloaded and tested them and they all give the same error.

Using equalizer APO version: 1.2.1, Windows 11 Pro, all the vsts/plugins tested was x64, x32 vsts/plugins does not work in Equalizer APO.
